pkg/base/test_server_args.go

```diff
@@ -169,6 +169,10 @@ type TestServerArgs struct {
 	// AutoConfigProvider provides auto-configuration tasks to apply on
 	// the cluster during server initialization.
 	AutoConfigProvider acprovider.Provider
+
+	// FastRangefeeds, if set, lowers the latency at which rangefeeds
+	// report changes.
```

Comment:
nit: changes are always emitted ~immediately, this knob only affects checkpoint lag and frequency.